core/transport/pom.xml

```diff
@@ -138,6 +138,7 @@
         <groupId>com.salesforce.servicelibs</groupId>
         <artifactId>proto-backwards-compatibility</artifactId>
         <configuration>
+          <allowBreakingChanges>true</allowBreakingChanges>
           <protoSourceRoot>${basedir}/src/main/proto</protoSourceRoot>
         </configuration>
```

Comment:
This change is not meant to be permanent. Just added this configuration to rename protobuf enum field, which should not break backward compatibility anyway.

Reply:
We need to revert this line before merging this PR because we want the proto lock to be the guard for compatibility.

Reply:
Does it mean we can't make the renaming of ServiceType? This pr currently needs this configuration to pass the build.

Reply:
Maybe not, what if you just add a new ServiceType and keep the old one?

Reply:
Updated. Now it builds without this line.